app/src/main/res/values/strings.xml

```diff
@@ -317,20 +317,20 @@
     <string name="account_about_subscribers">Subscribers:</string>
     <string name="account_about_description">Description:</string>
     <string name="account_about_joined">Joined:</string>
-    <string name="api_error">Something went wrong, please try later!</string>
+    <string name="api_error">Something went wrong, please try later.</string>
     <string name="action_bar_title_server_selection">Select Server</string>
 
     <string name="hello_blank_fragment">Hello blank fragment</string>
     <string name="action_bar_title_address_book" />
-    <string name="authentication_login_success">Login Successful</string>
-    <string name="authentication_login_failed">Login Failed!</string>
-    <string name="server_book_no_servers_found">Server Books is empty</string>
```

Comment:
Not sure what "Server Books" is.

Reply:
Server book is like an "address book", somewhere you list the instances you want to connect to.